WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

WARNING:py.warnings:/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/plum/type.py:207: UserWarning: Could not resolve the type hint of `numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]]`. I have ended the resolution here to not make your code break, but some types might not be working correctly. Please open an issue at https://github.com/wesselb/plum.
  return tuple(resolve_type_hint(arg) for arg in x)

rm: cannot remove 'checkpoint_sigma3': No such file or directory
rm: cannot remove 'checkpoint_sigma1': No such file or directory
rm: cannot remove 'checkpoint_sigma0': No such file or directory
rm: cannot remove 'checkpoint_sigma2': No such file or directory
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    model.load_state_dict(checkpoint["model_state_dict"])
    model.load_state_dict(checkpoint["model_state_dict"])
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 45]) from checkpoint, the shape in current model is torch.Size([15, 45]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 45]) from checkpoint, the shape in current model is torch.Size([15, 45]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    model.load_state_dict(checkpoint["model_state_dict"])
    model.load_state_dict(checkpoint["model_state_dict"])
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 55]) from checkpoint, the shape in current model is torch.Size([15, 55]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 55]) from checkpoint, the shape in current model is torch.Size([15, 55]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    model.load_state_dict(checkpoint["model_state_dict"])
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    model.load_state_dict(checkpoint["model_state_dict"])
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 65]) from checkpoint, the shape in current model is torch.Size([15, 65]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 65]) from checkpoint, the shape in current model is torch.Size([15, 65]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
Traceback (most recent call last):
  File "/home/nvs31/dlrbnicsx_cpu/demo/saddle_point_dlrbnicsx_distributed/dlrbnicsx_saddle_point_distributed.py", line 1464, in <module>
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    load_checkpoint(checkpoint_path_list_u[j], ann_model_list_u[j],
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/dlrbnicsx/interface/wrappers.py", line 121, in load_checkpoint
    model.load_state_dict(checkpoint["model_state_dict"])
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 75]) from checkpoint, the shape in current model is torch.Size([15, 75]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
    model.load_state_dict(checkpoint["model_state_dict"])
  File "/rds/user/nvs31/hpc-work/spack/install/linux-rocky8-icelake/gcc-11.3.0/python-3.11.6-d2mrkcs4zb3wcmdtrqu72dotvcqzwv2z/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for HiddenLayersNet:
	size mismatch for linear_layers.3.weight: copying a param with shape torch.Size([16, 75]) from checkpoint, the shape in current model is torch.Size([15, 75]).
	size mismatch for linear_layers.3.bias: copying a param with shape torch.Size([16]) from checkpoint, the shape in current model is torch.Size([15]).
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpiexec detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[56774,1],1]
  Exit code:    1
--------------------------------------------------------------------------
Command exited with non-zero status 1
	Command being timed: "mpiexec -n 8 python3 dlrbnicsx_saddle_point_distributed.py"
	User time (seconds): 211918.38
	System time (seconds): 719.57
	Percent of CPU this job got: 786%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 7:30:43
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 2305204
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 150620
	Minor (reclaiming a frame) page faults: 201128271
	Voluntary context switches: 48668397
	Involuntary context switches: 4567922
	Swaps: 0
	File system inputs: 1056392
	File system outputs: 9197112
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
